### PR TITLE
Fixes several broken links in Markdown files

### PR DESCRIPTION
**Pull request by sr-gi**
`clif/python/primer.md` contains several broken links to both examples and tests.

Did my best to cover every link, but there are some that point to files that seem to be missing (e.g. `third_party/car/embedded/common/python/can_protocol.clif` and `contentads/testing/drx/reporting/evenflow/python/sponge_uploader_clif_aux.h`) as well as some internal links to file sections, that are not supported by GH Markdown AFAIK (e.g. `renaming`, `unproperty`, ...).

`examples/README.md` also has a broken links pointing to `primer.md`

### CODE REVIEW

**Comment by sr-gi** [comment]
@googlebot I signed it!

**Comment by rwgk** [comment]
Hi Sergi, thanks for the fixes! I will manually transfer them over into the Google-internal source-of-truth repo (asap). We still have some work to do on the setup for reviewing and integrating PRs more automatically.

**Comment by sr-gi** [comment]
Hey Ralph, no worries.

Is there any alternative link where to access the processed HTML? I was trying to give CLIF a go but was quite hard to follow given the broken links.

**Comment by rwgk** [comment]
Hi Sergi, this wasn't as easy as I had hoped, but hopefully my commit fixes all the broken links you found.
(Not as easy because we have an internal version that's auto-edited for the github export; pretty messy.)
I will close this PR. Please let me now if there are still issues.
